docs/getting-started/downloads/index.hbs

```diff
@@ -0,0 +1,98 @@
+---
+title: Downloads
+description: Find the right way to include the AXA Web Toolkit into your app.
+nav: side/getting-started/downloads
+order: 2
+lunr: true
+toc:
+  - title: CSS and JS
+    slug: css-and-js
+  - title: Source files
+    slug: source-files
+  - title: CDN
+    slug: cdn
+  - title: Package managers
+    slug: package-managers
+fontUrl: https://fonts.googleapis.com/css?family=Source+Sans+Pro:400,600,700
+cssUrl: https://design.axa.com/toolkit/dist/all.css
```

Comment:
Totally understand your point. However, this happens to be the easiest way to include the Web Toolkit and experiment with it. Removing it will make it much harder to start with the toolkit.

How about we add a red alert saying that using the "CDN" is unstable and not ready yet except for experimentation?